evm/src/test/java/org/hyperledger/besu/evm/code/CodeV1Test.java

```diff
@@ -520,7 +520,12 @@ static Stream<Arguments> stackUnderflow() {
             "Stack underflow",
             "Operation 0x50 requires stack of 1 but may only have 0 items",
             0,
-            List.of(List.of("50 00", 0, 0x80, 1))));
+            List.of(List.of("50 00", 0, 0x80, 1))),
+        Arguments.of(
+            "double jumpi",
```

Comment:
Sorry, RJUMPI.  Code 3 line down is `PUSH0 PUSHO RJIMPI[5] PUSH0 PUSH0 RJUMPI[0] RETURN`.  The return could have either one stack item (if the first RJIMPI was taken) or two (if the first one was skipped and either of the second ones).  Return requires at least two stack items, so it could underflow.  Hence the validation error.